core/src/main/java/hudson/model/Computer.java

```diff
@@ -546,6 +547,16 @@ public int getNumExecutors() {
     }
 
     /**
+     * True if this computer is a Unix machine (as opposed to Windows machine).
+     *
+     * @return
+     *      null if the computer is disconnected and therefore we don't know whether it is Unix or not.
+     */
+    public Boolean isUnix() {
+        return !Functions.isWindows();
```

Comment:
Would have better made this `abstract` and implemented in `MasterComputer`. I doubt there are other `Computer` implementations, but if there are, this method implementation would probably be wrong—better to throw `AbstractMethodError`.